index.js

```diff
@@ -31,6 +31,10 @@ function waitConnection (socket, write) {
   }
 }
 
+function isExternalUrl (url = '') {
+  return urlPattern.test(url)
```

Comment:
Would you mind to check if `url.startsWith('http://') || url.startsWith('https://')` is faster than regex?